### PR TITLE
Implementation of #46

### PR DESCRIPTION
**Pull request by Waldleufer**
regarding #46

### CODE REVIEW

**Review by manifau — COMMENTED** [review]
The documentation made the code readable 👍

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -2,18 +2,20 @@ name: Backend CI
 
 on:
   push:
-    branches: [ main, backend-dev, release ]
+    branches: [ main, backend-dev-#46, release ]
```
Comment (Waldleufer) [inline .github/workflows/dotnet.yml:5]: This needs to be changed when merged again :) 

```suggestion
    branches: [ main, backend-dev, release ]
```

File: .github/workflows/dotnet.yml [hunk 2]
```diff
@@ -2,18 +2,20 @@ name: Backend CI
...
   pull_request:
-    branches: [ main, backend-dev, release ]
+    branches: [ main, backend-dev-#46, release ]
```
Comment (Waldleufer) [inline .github/workflows/dotnet.yml:7]:
```suggestion
    branches: [ main, backend-dev, release ]
```

File: .vs/ProjectSettings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (Waldleufer) [inline .vs/ProjectSettings.json:3]: What is this for? @SaiVarunVaranasi

**Review by Waldleufer — COMMENTED** [review]
I got some questions @SaiVarunVaranasi :) looking good else, thanks for looking into that!

File: .vs/ProjectSettings.json (hunk 3 above)
Comment (SaiVarunVaranasi) [reply .vs/ProjectSettings.json:3]: That got added automatically. I didn't make any changes to that file.

File: .github/workflows/dotnet.yml (hunk 1 above)
Comment (SaiVarunVaranasi) [reply .github/workflows/dotnet.yml:5]: Yes. That might fail again so I redirected it to the dev-#46 branch... during the sprint release when the changes are merged we can update that...

File: .vs/ProjectSettings.json (hunk 3 above)
Comment (Waldleufer) [reply .vs/ProjectSettings.json:3]: okay ^^

**Pull request merged**